ssntp/client.go

```diff
@@ -487,6 +487,7 @@ func (client *Client) UUID() string {
 // status frame or a CONFIGURE command one.
 func (client *Client) ClusterConfiguration() (payloads.Configure, error) {
 	var conf payloads.Configure
+	conf.Init()
```

Comment:
I don't think the client should be doing this, I think the configuration payload creator and sender (i.e. the scheduler) should send configuration data with proper default values. This is done here:
https://github.com/01org/ciao/blob/master/configuration/configuration.go#L50